pkg/csi/service/common/constants.go

```diff
@@ -248,4 +248,8 @@ const (
 	TriggerCsiFullSync = "trigger-csi-fullsync"
 	// CSIVolumeManagerIdempotency is the feature flag for idempotency handling in CSI volume manager
 	CSIVolumeManagerIdempotency = "improved-csi-idempotency"
+	// ImprovedVolumeTopology is the feature flag used to make the following improvements
+	// to topology feature:
+	// 1. Avoid taking in VC credentials in node daemonset.
```

Comment:
Do you have more? It looks like the comment is incomplete.

Reply:
Yes, the work for those tasks is not out, so I did not mention it here. I will add to this comment as and when the work gets checked in

Reply:
Ok, I'm fine with the code changes.